### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by yangfl**

### CODE REVIEW

File: src/engine/client/backend_sdl.cpp [hunk 1]
```diff
@@ -513,7 +513,7 @@ void CCommandProcessorFragment_OpenGL3_3::SetState(const CCommandBuffer::SState
 	}
 	else if(m_LastClipEnable)
 	{
-		//dont disable it always
+		//don't disable it always
```
Comment (Ryozuki) [inline src/engine/client/backend_sdl.cpp]: maybe you could even add a space before the comment, now that you are there :D

Comment (Ryozuki) [reply src/engine/client/backend_sdl.cpp]: and even capital letter

Comment (yangfl) [reply src/engine/client/backend_sdl.cpp]: but seems the existing codes is not using capital-letter comments...

Comment (Ryozuki) [reply src/engine/client/backend_sdl.cpp]: i still would do it. It's the correct format for comments, just change it where you edited the code.

**Comment by Ryozuki** [comment]
I think the remove trailing whitespaces should be in another pr and i also doubt such a big change would be merged.

**Comment by yangfl** [comment]
Should OK now

**Comment by heinrich5991** [comment]
Thanks.

bors r+

Generally, please only correct the style of code that you're currently touching anyway.

**Pull request merged**